iterations.r

```diff
@@ -1,69 +1,86 @@
-library(httr)
-library(jsonlite)
-library(ggplot2)
-library(DescTools)
-library(tidyverse)
-library(magrittr)
-library(rlang)
-library(lubridate)
-library(anytime)
-library(readr)
-library(yaml)
-
-#### 1: Beginning of script
-
-# Load function for posting GQL-queries and retrieving data: 
-source("functions/GQL_function.r")
-
-# The URL we will use is stored below: 
-
-configs <- 
-  read_yaml("vegvesen_configs.yml")
-
-
-gql_metadata_qry <- read_file("gql-queries/station_metadata.gql")
-
-# Let's try submitting the query: 
-
-stations_metadata <-
-  GQL(
-    query=gql_metadata_qry,
-    .url = configs$vegvesen_url
-    ) 
-
-
-#### 2: Transforming metadata
-
-source("functions/data_transformations.r")
-
-stations_metadata_df <- 
-  stations_metadata %>% 
-  transform_metadata_to_df(.)
-
-
-#### 3: Testing metadata
-source("functions/data_tests.r")
-test_stations_metadata(stations_metadata_df)
-
-
-### 5: Final volume query: 
-
-source("gql-queries/vol_qry.r")
```

Comment:
Make sure that the file is in the correct path. I had to change this path to run the script.

Reply:
I see now that this is also due to the fact that you did not change the existing file, but created a new one. But no worries, just keep it in mind for later :)